packages/core/src/snapshot.js

```diff
@@ -391,7 +392,7 @@ export function createSnapshotsQueue(percy) {
       return { ...snapshot, response };
     })
   // handle possible build errors returned by the API
-    .handle('error', (snapshot, error) => {
+    .handle('error', async (snapshot, error) => {
```

Comment:
In here we need some errors where we cannot connect percy - so we cannot call logs analysis either. 
it will be mostly found while create build or download of chromium and handle either here or in percy.js

Reply:
Resolved. Handled in `percy.js`